### PR TITLE
Log emag errors; Stop product notify if different affiliate id;

### PR DESCRIPTION
**Pull request by RaduGabor**

### CODE REVIEW

File: node/routes/productNotify.ts [hunk 1]
```diff
@@ -12,7 +12,14 @@ const LOG_TYPE = "productNotify";
 
 export async function productNotify(ctx: Context) {
   const { response, req, vtex } = ctx;
-  const body: { IdSku: string; ProductId: number } = await json(req);
+  const body: { IdSku: string; ProductId: number; IdAffiliate: string; } = await json(req);
+  const appSettings = await getAppSettings(vtex);
+  if (appSettings.affiliateId !== body.IdAffiliate) {
+    console.log(`Product ${body.ProductId} and SKU ${body.IdSku} has another affiliateId: ${body.IdAffiliate}. Required affiliateId: ${appSettings.affiliateId}`)
```
Comment (andreeacalinvtex) [inline node/routes/productNotify.ts:18]: Do we need this console.log here? I think we can use instead the VTEX logger option

File: node/routes/productNotify.ts [hunk 2]
```diff
@@ -12,7 +12,14 @@ const LOG_TYPE = "productNotify";
...
+    response.status = httpStatus.OK;
+    return { success: true };
+  }
+  console.log("PRODUCT NOTIFY", body, vtex);
```
Comment (andreeacalinvtex) [inline node/routes/productNotify.ts:22]: and here?

**Review by andreeacalinvtex — CHANGES_REQUESTED** [review]
Remove console logs

File: node/routes/productNotify.ts (hunk 1 above)
Comment (andreeacalinvtex) [inline node/routes/productNotify.ts:18]: Do we need this console.log here? I think we can use instead the VTEX logger option

**Comment by RaduGabor** [comment]
Done, added Vtex Logger

**Review by andreeacalinvtex — APPROVED** [review]

**Pull request merged**